Source/WebCore/dom/Element.cpp

```diff
@@ -1560,6 +1560,15 @@ int Element::clientHeight()
     return 0;
 }
 
+double Element::currentCSSZoom()
+{
+    protectedDocument()->updateLayoutIgnorePendingStylesheets({ LayoutOptions::ContentVisibilityForceLayout }, this);
```

Comment:
does this really need a layout?

Reply:
Due to changes in computed style.

Reply:
computed values don't required layout, used ones do.

Reply:
> computed values don't required layout, used ones do.

Nice! Updated and pushed commits. :-)

Reply:
@Ahmad-S792 I think `protectedDocument()->updateStyleIfNeeded()` should be enough here.